openhcl/underhill_mem/src/lib.rs

```diff
@@ -376,20 +339,22 @@ pub struct HardwareIsolatedMemoryProtector {
     inner: Mutex<HardwareIsolatedMemoryProtectorInner>,
     layout: MemoryLayout,
     acceptor: Arc<MemoryAcceptor>,
+    // TODO GUEST VSM: synchronize vtl protection bitmaps
```

Comment:
What's the idea here? Can we just use the same RCU mechanism, or are there more complications?